contribute/README.md

```diff
@@ -10,9 +10,13 @@ Thanks for enhancing OpenFisca anyway!
 
 ## Why contribute to OpenFisca?
 
-OpenFisca is a project being developed under the GPLv3 license or later.
+OpenFisca is a project being developed under the AGPL-3.0 license.
 The source code is freely available and modifiable.
 
+We ask for improvements and evolutions of the software to be shared back, for it allows the community to grow through improvements to the _core engine_ and updates to the _modeled tax-benefix systems (countries and cities)_.
+
+Although, we do not consider _extensions_ as being a form of derivative work, and as such you're free to license them as you wish.
```

Comment:
We can be more explicit:  _extensions_ or _reforms_

Reply:
I'm a little skeptical about differentiating the extension and the country package itself. 

If we take `france` as an example, why would contributors have to follow AGPL if they are working on a national benefit, while another contributor working on a Paris benefit would be exempted ? 

And if I really want to build a country model and avoid AGPL, with this policy I can just create a really minimal country model (just defining extensions) and a huge extension that contains all the code.

Reply:
> We can be more explicit: extensions or reforms

@benjello Why do you think _reforms_ shouldn't have a share-alike condition ?

> I'm a little skeptical about differentiating the extension and the country package itself.

@fpagnoux Fair enough. The objective is primarily to impose the sharing of the evolution of any country's modelled tax-benefit system. It makes sense then to extend the share-alike to the _extensions_.

Reply:
If you allow people not to share their extensions you should allow them not to share their reforms.
When people are working on their reform you cannot ask them to share unfinished work.
But if we take the route of strong sharing, when you distribute your reform we can ask people to share it.

Reply:
I did it extensively in the end.